docs/user/security/securing-communications/index.asciidoc

```diff
@@ -122,3 +125,33 @@ certificates, specify the location of the PEM file in the `kibana.yml` file:
 xpack.monitoring.elasticsearch.ssl.certificateAuthorities: /path/to/your/cacert.pem
 --------------------------------------------------------------------------------
 --
+
+[[extract-from-p12]]
+==== Extract files from a keystore
+
+If you created {es} and {kib} certificates with the {ref}/certutil.html[`elasticsearch-certutil`]
+utility as described in {ref}/configuring-tls.html#node-certificates[Generating Node Certificates],
+the output file is a PKCS#12 keystore. The keystore contains the private key,
+instance certificate, and CA certificate.  You can use `openssl` to extract the files
+from the keystore.
+
+Extract the private key for the instance certificate:
+
+[source,shell]
+----------------------------------------------------------
+openssl pkcs12 -in elastic-certificates.p12 -nocerts -nodes > server.key
+----------------------------------------------------------
+
+Extract the instance public certificate:
+
+[source,shell]
+----------------------------------------------------------
+openssl pkcs12 -in elastic-certificates.p12 -clcerts -nokeys > server.crt
```

Comment:
```suggestion
openssl pkcs12 -in elastic-certificates.p12 -clcerts -nokeys -out server.crt
```
These are almost the same, but they may function differently if the command errors out.